test/javascripts/components/chat-message-test.js

```diff
@@ -1,16 +1,64 @@
-import componentTest, {
-  setupRenderingTest,
-} from "discourse/tests/helpers/component-test";
+import User from "discourse/models/user";
+import { render } from "@ember/test-helpers";
+import ChatMessage from "discourse/plugins/discourse-chat/discourse/models/chat-message";
 import { exists, query } from "discourse/tests/helpers/qunit-helpers";
+import { setupRenderingTest } from "discourse/tests/helpers/component-test";
 import hbs from "htmlbars-inline-precompile";
 import ChatChannel from "discourse/plugins/discourse-chat/discourse/models/chat-channel";
-import EmberObject from "@ember/object";
 import I18n from "I18n";
-import { module } from "qunit";
+import { module, test } from "qunit";
 
 module("Discourse Chat | Component | chat-message", function (hooks) {
   setupRenderingTest(hooks);
 
+  function generateMessageProps(messageData = {}) {
```

Comment:
I swear this is living its last weeks

Reply:
Thank God! To quote Samuel L Jackson -- "Yes it deserves to die, and I hope it burns in hell!"